packages/components/src/components/input/input.stories.ts

```diff
@@ -69,731 +35,219 @@ export const Default = {
 };
 
 /**
- * Use the `label` attribute to give the input an accessible label. For labels that contain HTML, use the `label` slot instead.
+ * Use the `size` attribute to change the size.
  */
 
-export const Labels = {
-  args: overrideArgs([{ type: 'attribute', name: 'label', value: 'Label' }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Size = {
+  render: () => html`
+    <div class="grid grid-cols-2 gap-12 content-end">
+      <sd-input size="lg" label="Label"> </sd-input>
+      <sd-input size="md" label="Label"> </sd-input>
+      <sd-input size="sm" label="Label"> </sd-input>
+    </div>
+  `
 };
 
 /**
- * Add descriptive help text to an input with the `help-text` attribute. For help texts that contain HTML, use the `help-text` slot instead.
+ * Use the `label` attribute to give the input an accessible label.
```

Comment:
Use the "label" attribute to provide an accessible label for the input element.
Use the "label" slot to add custom markup.